src/providers/conversion/Coingecko.test.ts

```diff
@@ -0,0 +1,27 @@
+import { Currency } from './ConversionRateProvider';
+import { Coingecko } from './Coingecko';
+describe('Codegecko', () => {
+  let instance: Coingecko;
+  beforeEach(() => {
+    instance = new Coingecko();
+  });
+
+  it('getRate', async () => {
+    instance.load = () => {
```

Comment:
What about using a spy? 

https://github.com/metaplex/js/blob/main/test/actions/mintNFT.test.ts#L18

Reply:
If you prefer this way, I'll update test